internal/pkg/consensus/genesis.go

```diff
@@ -77,10 +77,16 @@ func ActorAccount(addr address.Address, amt abi.TokenAmount) GenOption {
 // MinerActor returns a config option that sets up an miner actor account.
```

Comment:
Changes here required due to: https://github.com/filecoin-project/specs-actors/pull/188